src/appleseed-max-impl/utilities.cpp

```diff
@@ -128,6 +129,29 @@ WStr replace_extension(const WStr& file_path, const WStr& new_ext)
     return new_file_path;
 }
 
+void update_map_buttons(IParamMap2* param_map)
+{
+    if (param_map == nullptr)
+        return;
+
+    IParamBlock2* param_block = param_map->GetParamBlock();
+
+    const int count = param_block->NumParams();
+    for (int i = 0; i < count; ++i)
```

Comment:
An elegant idiom we use all the time is this:

    for (int i = 0, e = param_block->NumParams(); i < e; ++i)

(`e` for "end")

You don't have to use it, but it's good to know it.

Reply:
Yes! Now I remember you told me that before. But I couldn't remember by myself. Thanks!